name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/consorsbank/ConsorsbankPDFExtractorTest.java

```diff
@@ -325,16 +325,12 @@ public void testErtragsgutschrift10() throws IOException
         assertThat(t.getSecurity().getName(), is("OMNICOM GROUP INC. Registered Shares DL -,15"));
         assertThat(t.getSecurity().getIsin(), is("US6819191064"));
         assertThat(t.getSecurity().getWkn(), is("871706"));
-        assertThat(t.getSecurity().getCurrencyCode(), is(CurrencyUnit.USD));
+        assertThat(t.getSecurity().getCurrencyCode(), is(CurrencyUnit.EUR));
```

Comment:
I am not sure why we should change the currency of the created security from USD to EUR.

If you look at the extracted text, it is clear that the security is in USD:

```
OMNICOM GROUP INC. Registered Shares DL -,15 871706 US6819191064
Bestand
25 Stück
Dividende pro Stück 0,60 USD Schlusstag 17.12.2017
Brutto in USD 15,00 USD
```

Reply:
I rolled back the changes to the importer and the corresponding test case.